commands/host/phpmyadmin

```diff
@@ -5,6 +5,11 @@
 ## Usage: phpmyadmin
 ## Example: "ddev phpmyadmin"
 
+if [ "${DDEV_PROJECT_STATUS-running}" != "running" ]; then
```

Comment:
I like the technique, it's a good one. But it should be like this, right?

```suggestion
if [ "${DDEV_PROJECT_STATUS:-running}" != "running" ]; then
```

Reply:
No, this can give an unexpected result for the case when `$DDEV_PROJECT_STATUS` is defined but empty.

bad:
```
DDEV_PROJECT_STATUS="" && echo "status=${DDEV_PROJECT_STATUS:-running}"
status=running
```

good:
```
DDEV_PROJECT_STATUS="" && echo "status=${DDEV_PROJECT_STATUS-running}"
status=
```

Reply:
POSIX: Parameter Expansion Examples: https://stackoverflow.com/a/16753536/8097891

Reply:
I tested this behavior on macOS and it works the same way.
I'm going to merge `${DDEV_PROJECT_STATUS-running}"` because it works exactly as it should.